schema/geometry_schema.json

```diff
@@ -94,6 +94,38 @@
 						},
 						"north_axis": {
 							"type": "number"
+						},
+						"ground": {
+							"type": "object",
+							"properties": {
+								"floor_offset": {
+									"description": "Specifies the vertical offset between the building floor and the ground plane. Use a value < 0 for buildings with below-grade spaces, or a value > 0 for, e.g., buildings on piers.",
+									"type": "number",
+									"default": 0
+								},
+								"azimuth_angle": {
+									"description": "Direction of the ground plane, in clockwise degrees from the positive y-axis when viewed from above.",
+									"type": "number"
+								},
+								"tilt_angle": {
+									"description": "Angle of the ground plane, in degrees, from horizontal.",
+									"type": "number",
+									"minimum": 0,
+									"maximum": 90,
+									"exclusiveMaximum": true,
+									"default": 0
+								},
+								"x_shift": {
```

Comment:
I'm not really crazy about these members, what are they here for?  Seems like it might be easier to just let the user repick a new x, y origin at any time using a special tool.